index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+<main id="main-doc">
+  <section class="main-section" id="Section_1">
+    <header>
+      <p>Section1</p>
+      <p>About <code>this</code>  section</p>
+      <li>Javascript</li>
```

Comment:
li tag usually comes with ul tag. Please change it accordingly to create an unordered list.